pom.xml

```
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+ Copyright (C) 2024 eXo Platform SAS.
+
+ This is free software; you can redistribute it and/or modify it
+ under the terms of the GNU Lesser General Public License as
+ published by the Free Software Foundation; either version 2.1 of
+ the License, or (at your option) any later version.
+
+ This software is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this software; if not, write to the Free
+ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>addons-exo-parent-pom</artifactId>
+        <groupId>org.exoplatform.addons</groupId>
+        <version>18-M01</version>
+    </parent>
+
+    <groupId>org.exoplatform.addons.external-visio-connector</groupId>
+    <artifactId>external-visio-connector</artifactId>
+    <version>1.0.x-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>eXo Add-on:: eXo External Visio Connector</name>
+    <description>eXo Web Conferencing portal extension</description>
+    <properties>
+      <addon.exo.web-conferencing.version>2.6.x-SNAPSHOT</addon.exo.web-conferencing.version>
```

```suggestion
      <addon.exo.web-conferencing.version>2.5.x-SNAPSHOT</addon.exo.web-conferencing.version>
```

Addon should be firstly done on 6.5.x